nbconvert/preprocessors/tests/test_execute.py

```diff
@@ -510,6 +515,65 @@ def test_busy_message(self, preprocessor, cell_mock, message_mock):
         # Ensure no outputs were generated
         assert cell_mock.outputs == []
 
+    @prepare_cell_mocks({
```

Comment:
These all look great!

Reply:
All the tests are great work by @MSeal (thank you!) it's my first contribution so I was unfamiliar with how to use these test frameworks.